config/examples/max_allowed_days.yml

```diff
@@ -0,0 +1,14 @@
+max_allowed_days: &default
```

Comment:
I am reluctant to create a file for each features we implement or will be implementing
Though `config/settings.yml` is now a mess... Any idea of how to solve this dilemna?

Reply:
For me we need a way to don't put almost absolutely everything inside `settings.yml` because that is a complete mess and imo a separate file for this separate feature with some related attributes and specific behaviour in common makes more sense to me. But there can be better ideas and I'm opened to hear them 😄

Reply:
Decision for now: https://github.com/3scale/porta/pull/574#discussion_r254241558